src/main/java/seedu/address/logic/commands/patient/DeletePatientCommand.java

```diff
@@ -5,51 +5,56 @@
 import java.util.List;
 
 import seedu.address.commons.core.Messages;
-import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.patient.Patient;
 
 /**
- * Deletes a patient identified using it's displayed index from the address book.
+ * Deletes a patient identified by the patient's name from the application.
```

Comment:
Because `isSamePatient(...)` uses name, age, and phone number as criteria to detect duplicate `Patient` in a list, will this not pose a problem when deleting? It is possible to have more than 1 `Patient` with the same name after all. In that case, only the first matching `Patient` is deleted. 

I think it may be a little strange that the user cannot select which `Patient` to delete if they share the same name.  I am not sure about the optimal way to solve this problem. Would deleting by index be better?

Reply:
should we make it such that there cannot be patient with the same name?

Reply:
I agree with Ming De I think we should stick to making sure patient does not have the same name. If we delete by index, in the case of 1000 entries, the user may have to scroll too much to find the patient (using searchpatient command doesn't show the index also :/). 
If we just ensure patient cannot have the same name, then all we need to change is the isSamePatient(...) function (to guard against duplicate patient being entered). If we need to check for the age and phone number identifiers, then we may need massive changes for deletepatient and editpatient command :/.
Well, we can justify that the odds of patients having the same name is really very slim haha (https://www.quora.com/What-is-the-probability-of-two-people-having-the-same-name-and-also-living-in-the-same-city)